src/features/pastelStatistics/components/card/Card.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { useHistory } from 'react-router-dom'
+
+import bg1 from '../../assets/images/bg1.png'
+import bg2 from '../../assets/images/bg2.png'
+import bg3 from '../../assets/images/bg3.png'
+import bg4 from '../../assets/images/bg4.png'
+import bg5 from '../../assets/images/bg5.png'
+import bg6 from '../../assets/images/bg6.png'
+import bg7 from '../../assets/images/bg7.png'
+import bg8 from '../../assets/images/bg8.png'
+import bg9 from '../../assets/images/bg9.png'
```

Comment:
These images are pretty low-resolution. I can see they are pixelated when I run this branch. Please improve their quality

Reply:
I'm getting these images from screenshot of running app.
I am not sure how can improve these images.
Maybe we need help from designer.

Reply:
Just don't optimize them too much